MetaMask/metamask-extension
# chore: updated celo logo

## **Description**



This PR updates the Celo network icon from a transparent black logo to one featuring a yellow background for improved recognizability.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/29047?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this select network button
2. Select Celo Network
3. Celo Network should have logo with yellow background (better accessibility)

## **Screenshots/Recordings**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
